docs/user_guide/ip_cores/axi_adc/index.rst

```diff
@@ -219,6 +218,60 @@ see :ref:`axi_adc adc-channel` section.
    To find the instantiation of this module search for ``up_adc_channel`` inside
    the IP's directory.
 
+.. _generic-adc-register-access:
+
+Register access
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ADC IP supports **16 channels**, numbered from **0 to 15**. The **base
+registers** start at offset ``0x0`` and the **common (global) registers** start
```

Comment:
I would add here a "*" and give more details below.
Usually the start offset is 0x0, But it can be something lese like in the case of axi_adrv9001 which has two adc_common register spaces. 
Same for DAC. It can be 0x0 or something in the interval 0x100 to 0x3F00.